### DISCUSSION TITLE
Black-Litterman Entropy Pooling

### DISCUSSION THREAD

**Discussion by ActurialCapital** [dcajasn/Riskfolio-Lib#135] [Ideas]
@dcajasn,

This method could be particularly useful when managing a top-down strategy and is of interest of many practitioners in the industry (as of today). Below is an abstract:

1. The Black-Litterman model is a mathematical framework for combining subjective views or forecasts with objective market data in order to generate improved portfolio weight estimates. 
2. The entropy pooling approach is a specific method within the Black-Litterman framework for combining multiple views, where each view is represented as a ranking of assets rather than as a specific weighting.
3. In the entropy pooling approach, the subjective views are transformed into probability distributions over the set of assets, with the probability of each asset being proportional to its rank in the view. 
4. These probability distributions are then combined using a process called entropy pooling, which involves taking the weighted average of the individual probability distributions while minimizing the amount of uncertainty or "entropy" in the resulting combined distribution.
5. The resulting combined probability distribution can then be used to generate  improved portfolio weight estimates by solving a optimization problem that takes into account both the market data and the subjective views. The optimization problem can be formulated as a quadratic programming problem and solved using standard optimization techniques.

Let me know what you think.

**Comment by antonvorobets** [comment]
Check out

 https://github.com/fortitudo-tech/fortitudo.tech 

or


https://github.com/fortitudo-tech/entropy-pooling
